app-tasks/rf/src/rf/commands/find_aoi_projects.py

```diff
@@ -0,0 +1,60 @@
+import logging
+import os
+import subprocess
+
+import click
+import boto3
+
+from ..utils.batch import get_latest_job_revision
+
+logger = logging.getLogger(__name__)
+
+ENVIRONMENT = os.getenv('ENVIRONMENT').title()
+AOI_UPDATE_BATCH_JOB_NAME = os.getenv('AOI_UPDATE_BATCH_JOB_NAME')
```

Comment:
I don't know that we have a solid convention for these names yet, but it would be great to prefix all of the Batch variables with `BATCH_` or `AWS_BATCH_`.

Reply:
I'll switch to `AWS_BATCH_` -- that's the most explicit naming